engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ValidationContext.scala

```diff
@@ -15,6 +17,8 @@ case class ValidationContext(localVariables: Map[String, TypingResult] = Map.emp
                              globalVariables: Map[String, TypingResult] = Map.empty,
                              parent: Option[ValidationContext] = None) {
 
+  val VariableNamePattern: Pattern = "^[a-zA-Z0-9_]*$".r.pattern
```

Comment:
Maybe we should use javax.lang.model.SourceVersion.isIdentifier? e.g. identifiers cannot start with digits?